tests/unit-tests/test_config.py

```diff
@@ -1177,14 +1177,26 @@ def create_gnn_config(tmp_path, file_name):
     with open(os.path.join(tmp_path, file_name+"4.yaml"), "w") as f:
         yaml.dump(yaml_object, f)
 
+    yaml_object["gsf"]["node_classification"] = {}
+    yaml_object["gsf"]["basic"] = {
+        "model_encoder_type": "rgat"
+    }
+    yaml_object["gsf"]["gnn"] = {
+        "node_feat_name": ["ntype0:feat_name, feat_name2 ", "ntype1: fname"],
```

Comment:
Do we have a test case like: `"node_feat_name": ["ntype0:feat_name,feat_name2 ", "ntype1: fname"]` ?

Reply:
Not yet, this is the first one.

Reply:
Yes, that is test case 4.